src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.container,
+.navigation,
+.logo,
+.link,
+.box {
+  display: flex;
+}
+
+.navigation {
+  margin-right: 50px;
+}
+
+.box {
+  margin-right: 20px;
+}
+
+.box:last-child {
+  margin-right: 0;
+}
+
+.container {
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.link {
+  height: 60px;
```

Comment:
it is better to set the height in the outer container, now the height of your container is 64px when should be 60px